internal/cli/atlas/privateendpoints/aws/interfaces/create.go

```diff
@@ -39,7 +39,7 @@ func (opts *CreateOpts) initStore() error {
 	return err
 }
 
-var createTemplate = "Interface endpoint '{{.ID}}' created.\n"
+var createTemplate = "Interface endpoint '{{.InterfaceEndpointID}}' created.\n"
```

Comment:
[q] is this change correct? can you please share the example? also if it's correct why only for AWS and not azure?

Reply:
```
./bin/mongocli atlas privateendpoint aws interface create 603e453dda476d77568279e8 --privateEndpointId vpce-0fef3160861c9e8e0
Interface endpoint 'vpce-0fef3160861c9e8e0' created.
```
The response is different.


AWS:
```json
{
  "connectionStatus": "PENDING",
  "deleteRequested": false,
  "errorMessage": null,
  "interfaceEndpointId": "vpce-08fb7e9319909ec7b"
}
```


Azure:

```json
{
  "deleteRequested": false,
  "errorMessage": null,
  "privateEndpointConnectionName": null,
  "privateEndpointIPAddress": "10.0.0.4",
  "privateEndpointResourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/privatelink/providers/Microsoft.Network/privateEndpoints/test",
  "status": "INITIATING"
}
```

I forgot to use `PrivateEndpointResourceID` for azure though